Makefile

```diff
@@ -715,7 +715,7 @@ $(OPERATOR_SDK):
 	chmod +x $(OPERATOR_SDK)
 
 ISTIOCTL = $(shell pwd)/bin/istioctl
-ISTIOCTL_VERSION = 1.17.2
+ISTIOCTL_VERSION = 1.23.3
```

Comment:
What's the difference between these two version?

Reply:
The new version fixed some iptables issues I have in my local environment. I believe the new version will be more stable.